dds_cli/utils.py

```diff
@@ -190,6 +190,19 @@ def perform_request(
     # Check if response is ok.
     if not response.ok:
         message = error_message
+
+        if response.status_code == http.HTTPStatus.BAD_REQUEST:
+            if DDSEndpoint.CREATE_PROJ in endpoint:
+                message += f": {__project_creation_error(response_json)}"
+
+            raise dds_cli.exceptions.DDSCLIException(message=message)
+
+        if response.status_code == http.HTTPStatus.FORBIDDEN:
+            if DDSEndpoint.USER_ADD in endpoint:
+                message += ": You do not have the required permissions to create a project."
+
+            raise dds_cli.exceptions.DDSCLIException(message=message)
```

Comment:
Do you mean `DDSEndpoint.CREATE_PROJ`? The `DDSEndpoint.USER_ADD` endpoint is not called from the CLI when we try to add users while creating a project. The project-creation endpoint handles that by calling the user-add endpoint but it's still the create project endpoint that's called here.